Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/grenade.yml

```
@@ -7,7 +7,7 @@
   - type: Sprite
     sprite: Objects/Weapons/Guns/Projectiles/projectiles2.rsi
     layers:
-    - state: buckshot
+    - state: grapeshot
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 5

(indentation)